create_aruco_dict.py

```diff
@@ -0,0 +1,25 @@
+import sys
+sys.path.append('/usr/local/lib/python2.7/site-packages')
+import cv2
+import numpy as np
+import cv2.aruco as aruco
+
+#small_dict = aruco.generateCustomDictionary(6, 3)
+NUM_OF_MARKERS = 6
+small_dict = aruco.Dictionary_create(NUM_OF_MARKERS, 3)
+'''
+    drawMarker(...)
+        drawMarker(dictionary, id, sidePixels[, img[, borderBits]]) -> img
+'''
+             
+print(small_dict)
```

Comment:
switch to `logging.<LOG_LEVEL>`? Also, why is this here?

Reply:
its mostly legacy from debugging, as with the later print. I don't think logging's needed here, since it should just run once